fetch.js

```diff
@@ -236,6 +236,21 @@
       }
     }
 
+    if (support.arrayBuffer) {
+      this.arrayBuffer = function() {
+        if (this._bodyArrayBuffer) {
+          var rejected = consumed(this)
+          if (rejected) {
+            return rejected
```

Comment:
It's fine to not match the style of others in this instance. Other invocations of `consumed()` can not efficiently be condensed into a one-liner — except for `text()`, which already uses a ternary operator.